### ISSUE TITLE
Display warning for excessive friendly fire

### ISSUE THREAD

**Issue by mithrand0** [ReactiveDrop/reactivedrop_public_src#408]
Users join games, some even expect death match. They kill/harm others, but don't see the issue with that right away. When a user does such excessive damage in a game, it would be good to display a warning on the screen, that he is doing so. Something like the game instructor, but then forced.

> You are hurting other team members, try to not hit your team. You can also consider different weapons.
> You are burning your team members, use the extinguisher to put out the fire or consider a different weapon.

**Comment by geotavros** [comment]
Game instructor already does a great job explaining this. There's a hint "Don't damage your team mates".

Do you mean to additionally print the messages in chat or showing a messagebox?

**Comment by mithrand0** [comment]
> Do you mean to additionally print the messages in chat or showing a messagebox?

Yes. This to also catch the scenario when people having the instructor disabled.

**Comment by wawawawawawawa** [comment]
That message (pls no OK box xD) will work on weapons that don't one hit, albeit I don't think it will do much.

For gl, or pellet based like shotgun/vindi, the increased damage from successive ff will pretty much one shot.
For flamers or rifles, maybe, and even then, doubt it will matter much for the solo speedrunner

Also, we might need some hint "Don't go inside chainsaw" as well haha

Seriously though, I think there should be some "ASBI FF" modifier, with some achievement related to it (like the Hardcore achievement) to help player learn about friendly fire/position.
